package.json

```diff
@@ -10,8 +10,7 @@
     "test-local": "BABEL_ENV=test npm run jest -- --watch",
     "build": "BABEL_ENV=build webpack",
     "watch": "BABEL_ENV=build webpack -w",
-    "prepublishOnly": "BABEL_ENV=build webpack -p && npm run generate-actions",
-    "generate-actions": "jsdoc2md --files lib/**/actions.js > docs/vuex-actions.md"
+    "prepublishOnly": "BABEL_ENV=build webpack -p"
```

Comment:
Hmmm, I think we still need the `generate-actions` script to automatically create docs for the vuex actions.

Reply:
Nice catch